sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Microsoft.Azure.WebPubSub.AspNetCore.csproj

```diff
@@ -6,14 +6,14 @@
     <PackageTags>Azure, WebPubSub</PackageTags>
     <RequiredTargetFrameworks>netcoreapp3.1</RequiredTargetFrameworks>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <NoWarn>$(NoWarn);419</NoWarn>
+    <NoWarn>$(NoWarn);419;AZC0001;AZC0011</NoWarn>
```

Comment:
We need to remove 'AZC0011' from here too.
This project has IVT `Microsoft.Azure.WebJobs.Extensions.WebPubSub`